src/devices/bus/pc_kbd/keyboards.cpp

```diff
@@ -34,6 +34,11 @@ void pc_at_keyboards(device_slot_interface &device)
 	device.option_add(STR_KBD_CHERRY_G80_1500, CHERRY_G80_1500);
 }
 
+void pc_ps2_keyboards(device_slot_interface &device)
+{
+	device.option_add(STR_KBD_MICROSOFT_NATURAL, PC_KBD_MICROSOFT_NATURAL);
+}
+
```

Comment:
Do the other AT keyboards not work with a PS/2?  They’re electrically compatible and support at least some of the same scan code sets.

Reply:
Some do, some don't. I just have this one in so it defaults to one that works.